Marlin/pins.h

```diff
@@ -308,7 +308,12 @@
 #define _E3_PINS
 #define _E4_PINS
 
-#if EXTRUDERS > 1
+#if ENABLED (SINGLENOZZLE_SWITCHING_EXTRUDER) // needs only one stepper for 2 Extruder
+  #if EXTRUDERS > 2
```

Comment:
Since `SINGLENOZZLE_SWITCHING_EXTRUDER` uses exactly 2 or 4 extruders, then this block should actually be:
```cpp
#if ENABLED(SINGLENOZZLE_SWITCHING_EXTRUDER)
  #undef _E1_PINS
  #define _E1_PINS E1_STEP_PIN, E1_DIR_PIN, E1_ENABLE_PIN, E1_MS1_PIN, E1_MS2_PIN,
  #if EXTRUDERS == 4
    #undef _E2_PINS
    #define _E2_PINS E2_STEP_PIN, E2_DIR_PIN, E2_ENABLE_PIN, E2_MS1_PIN, E2_MS2_PIN,
    #undef _E3_PINS
    #define _E3_PINS E3_STEP_PIN, E3_DIR_PIN, E3_ENABLE_PIN, E3_MS1_PIN, E3_MS2_PIN,
  #endif
#elif EXTRUDERS > 1
  . . .
#endif
```